presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -147,6 +147,17 @@ Accessors
     Returns the 2-dimensional cartesian minimum distance (based on spatial ref)
     between two geometries in projected units.
 
+.. function:: ST_GeometryN(Geometry, index) -> Geometry
+
+    If the geometry is a collection of geometries (e.g., GEOMETRYCOLLECTION or MULTI*),
+    returns the geometry at a given index.
+    Indexes start with one.
+    If the given index is less than 1 or greater then the total number of elements in the collection,
+    returns null.
+    Use :func:ST_NumGeometries to find out the total number of elements.
+    Singular geometries, POINT, LINESTRING, POLYGON, are treated as collections of one element.
```

Comment:
Either say `, such as POINT, LINESTRING, and POLYGON` or `(e.g., POINT, LINESTRING, and POLYGON)` as you do above.